### ISSUE TITLE
Issue 0.10.0 Beta 8 Retain modes

### ISSUE THREAD

**Issue by AlexanderSch90** [blakeblackshear/frigate#2756]
Hello, I suspect I have found a bug here in the current beta. 

In your example with the current release the block "Record" can look like this:
```
record:
  enabled: True
  retain:
    days: 7
    mode: motion
  events:
    retain:
      default: 14
      mode: active_objects
```

Mine looks like this:
```
record:
  enabled: True
  retain:
    days: 1
    mode: motion
  events:
    max_seconds: 300
    pre_capture: 5
    post_capture: 5
    objects:
      - person
      - car
      - dog
    required_zones: []
    retain:
      default: 20
      mode: active_objects
      objects:
        dog: 2
```

I get the following log entry:

```
[2022-02-06 08:40:17] frigate.app                    INFO    : Starting Frigate (0.10.0-b1e84ca)
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
[2022-02-06 08:40:17] frigate.config                 WARNING : Recording retention is configured for motion and event retention is configured for active_objects. The more restrictive retention policy will be applied.
Starting migrations
```

Probably per camera (I have 8 pieces) once the message...

**Comment by blakeblackshear** [comment]
It logs per camera. I need to clarify the message.

**Comment by AlexanderSch90** [comment]
Fixed with 010.0 Beta 9!
